src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockDefault.java

```diff
@@ -76,13 +80,13 @@ void method5(int a, int b) {
             case 1:
             case 2:
             case 3:
-            default:       // violation
+            default:        // violation on next line
```

Comment:
It is good to be consistent, I would make all comments that cannot be placed on the same line identical.

Reply:
Done.

Reply:
Resolved.